src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -80,31 +91,35 @@ public static Confidence.Builder builder(String clientSecret) {
 
   public static class Builder {
     private final String clientSecret;
-    private ManagedChannel managedChannel;
+    private ManagedChannel flagResolverManagedChannel =
+        ManagedChannelBuilder.forAddress("edge-grpc.spotify.com", 443).build();
```

Comment:
Adding the address as a string seems error-prone - should we add is as a const?

Reply:
@nickybondarenko if the string is used once then I believe it's unnecessary to create a const.

Reply:
but it seems like we are also using the same host and port in grpc uploader [here](https://github.com/spotify/confidence-openfeature-provider-java/pull/92/files#diff-93e4c877f3d0f166ae56346cc38e4e67c4728e8cb355255d7c744606737dd344R119)

Reply:
I removed the other occurences since the user should never have to construct the grpc-uploader manually.